speaksfer/settings/base.py

```diff
@@ -0,0 +1,119 @@
+"""
+Django settings for speaksfer project.
+Generated by 'django-admin startproject' using Django 4.0.5.
+For more information on this file, see
+https://docs.djangoproject.com/en/4.0/topics/settings/
+For the full list of settings and their values, see
+https://docs.djangoproject.com/en/4.0/ref/settings/
+"""
+
+from pathlib import Path
+
+import dj_database_url
+from decouple import config
+
+# Build paths inside the project like this: BASE_DIR / 'subdir'.
+BASE_DIR = Path(__file__).resolve().parent.parent
+
+
+# Quick-start development settings - unsuitable for production
+# See https://docs.djangoproject.com/en/4.0/howto/deployment/checklist/
+
+# SECURITY WARNING: keep the secret key used in production secret!
+SECRET_KEY = config("SECRET_KEY")
+
+# SECURITY WARNING: don't run with debug turned on in production!
+DEBUG = False
+
+ALLOWED_HOSTS = ["localhost"]
```

Comment:
remove "localhost" , leave it as an empty list

Reply:
Okay. On it